eureka-client/src/main/java/com/netflix/discovery/DiscoveryEvent.java

```diff
@@ -3,31 +3,11 @@
  */
 package com.netflix.discovery;
 
-
 /**
  * Class to be extended by all discovery events. Abstract as it
  * doesn't make sense for generic events to be published directly.
  * 
- * @author brenuart
+ * @deprecated 2016-02-15 Extend from {@link AbstractEurekaEvent} instead
  */
-public abstract class DiscoveryEvent {
-
-	/** System time when the event happened */
-	private final long timestamp;
-	
-	
-	/**
-	 * Create a new DiscoveryEvent
-	 */
-	public DiscoveryEvent() {
-		super();
-		this.timestamp = System.currentTimeMillis();
-	}
-	
-	/**
-	 * Return the system time in milliseconds when the event happened.
-	 */
-	public final long getTimestamp() {
-		return this.timestamp;
-	}
+public abstract class DiscoveryEvent extends AbstractEurekaEvent {
```

Comment:
If the aim of this is to move from discovery* to eureka* in terms of naming, then I would propose that it's fine to keep the old event model as there are functionally no difference. We already have .discovery. in our package names and it would introduce more api complexity with the renaming, than keeping with the current.